src/Bridge/SimpleCache/SimpleCacheBridge.php

```diff
@@ -0,0 +1,188 @@
+ * with this source code in the file LICENSE.
+ */
+
+namespace Cache\Bridge\SimpleCache;
+
+use Cache\Bridge\SimpleCache\Exception\InvalidArgumentException;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Psr\Cache\InvalidArgumentException as CacheInvalidArgumentException;
+use Psr\SimpleCache\CacheInterface;
+
+class SimpleCacheBridge implements CacheInterface
+{
+    /**
+     * @type CacheItemPoolInterface
+     */
+    protected $cacheItemPool;
+
+    /**
+     * SimpleCacheBridge constructor.
+     */
+    public function __construct(CacheItemPoolInterface $cacheItemPool)
+    {
+        $this->cacheItemPool = $cacheItemPool;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function get($key, $default = null)
+    {
+        try {
+            $item = $this->cacheItemPool->getItem($key);
+        } catch (CacheInvalidArgumentException $e) {
+            throw new InvalidArgumentException($e->getMessage(), $e->getCode(), $e);
+        }
+
+        if (!$item->isHit()) {
+            return $default;
+        }
+
+        return $item->get();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function set($key, $value, $ttl = null)
+    {
+        try {
+            $item = $this->cacheItemPool->getItem($key);
+        } catch (CacheInvalidArgumentException $e) {
+            throw new InvalidArgumentException($e->getMessage(), $e->getCode(), $e);
+        }
+
+        $item->set($value);
+        $item->expiresAfter($ttl);
+
+        return $this->cacheItemPool->save($item);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function delete($key)
+    {
+        try {
+            return $this->cacheItemPool->deleteItem($key);
+        } catch (CacheInvalidArgumentException $e) {
+            throw new InvalidArgumentException($e->getMessage(), $e->getCode(), $e);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function clear()
+    {
+        return $this->cacheItemPool->clear();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getMultiple($keys, $default = null)
+    {
+        if (!is_array($keys)) {
+            if (!$keys instanceof \Traversable) {
+                throw new InvalidArgumentException('$keys is neither an array nor Traversable');
+            }
+
+            // Since we need to throw an exception if *any* key is invalid, it doesn't
+            // make sense to wrap iterators or something like that.
+            $keys = iterator_to_array($keys);
+        }
+
+        try {
+            $items = $this->cacheItemPool->getItems($keys);
+        } catch (CacheInvalidArgumentException $e) {
+            throw new InvalidArgumentException($e->getMessage(), $e->getCode(), $e);
+        }
+
+        foreach ($items as $key => $item) {
+            /** @type $item CacheItemInterface */
+            if (!$item->isHit()) {
+                yield $key => $default;
```

Comment:
You must do `continue;` after here or the next yield will be executed next time..